dora/shell/src/main/java/alluxio/cli/fsadmin/report/MetricsCommand.java

```diff
@@ -58,47 +59,14 @@ public MetricsCommand(MetricsMasterClient metricsMasterClient, PrintStream print
    * @return 0 on success, 1 otherwise
    */
   public int run() throws IOException {
-    mMetricsMap = mMetricsMasterClient.getMetrics();
-    SortedSet<String> names = new TreeSet<>(mMetricsMap.keySet());
-    for (String name : names) {
-      if (!isAlluxioMetric(name)) {
-        continue;
-      }
-      MetricValue metricValue = mMetricsMap.get(name);
-      String strValue;
-      if (metricValue.hasStringValue()) {
-        strValue = metricValue.getStringValue();
-      } else {
-        double doubleValue = metricValue.getDoubleValue();
-        if (name.contains(BYTES_METRIC_IDENTIFIER)) {
-          // Bytes long can be transformed to human-readable format
-          strValue = FormatUtils.getSizeFromBytes((long) doubleValue);
-          if (name.contains(THROUGHPUT_METRIC_IDENTIFIER)) {
-            strValue = strValue + "/MIN";
-          }
-        } else if (DoubleMath.isMathematicalInteger(doubleValue)) {
-          strValue = DECIMAL_FORMAT.format((long) doubleValue);
-        } else {
-          strValue = String.valueOf(doubleValue);
-        }
-      }
-      mPrintStream.printf(INFO_FORMAT, name, metricValue.getMetricType(), strValue);
+    ObjectMapper objectMapper = new ObjectMapper();
+    MetricsOutput metricsInfo = new MetricsOutput(mMetricsMasterClient.getMetrics());
```

Comment:
yea this should be the right way, you call the client to get the info then pass to the object wrapper (MetricsOutput) purely for some conversion and display